physical/physical.go

```diff
@@ -88,6 +88,7 @@ var BuiltinBackends = map[string]Factory{
 	"dynamodb":  newDynamoDBBackend,
 	"etcd":      newEtcdBackend,
 	"mysql":     newMySQLBackend,
+	"postgres":  newPostgreSQLBackend,
```

Comment:
I could be persuaded either way on this. I used "postgresql" in most places because I'm more fastidious than I probably need to be. If it's left as "PostgreSQLBackend", I should probably change this line  to "postgresql" as well.